src/Microsoft.Health.Fhir.Core/Messages/Get/GetOperationVersionsResponse.cs

```diff
@@ -0,0 +1,26 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
+using EnsureThat;
+
+namespace Microsoft.Health.Fhir.Core.Messages.Get
+{
+    public class GetOperationVersionsResponse
+    {
+        public GetOperationVersionsResponse(List<string> supportedVersions, string defaultVersion)
+        {
+            EnsureArg.IsNotNull(supportedVersions, nameof(supportedVersions));
+            EnsureArg.IsNotNullOrWhiteSpace(defaultVersion, nameof(defaultVersion));
+
+            SupportedVersions = supportedVersions;
+            DefaultVersion = defaultVersion;
+        }
+
+        public List<string> SupportedVersions { get; }
```

Comment:
Also `IList<string>`

Reply:
Changing this here requires me to change it in VersionsResult.cs, and the XML serializer doesn't seem to like IList very much (the property will be ignored and will not show up in the output). Is there a better solution than converting from IList to List in VersionsResult?

Reply:
Ok, lets leave it like this for now